server/game/cards/20-HMW/Jinglebell.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../drawcard.js');
+const GameActions = require('../../GameActions');
+
+class Jinglebell extends DrawCard {
+    setupCardAbilities() {
+        this.interrupt({
+            when: {
+                onCharacterKilled: event => event.card === this
+            },
+            handler: context => {
+                this.game.resolveGameAction(
+                    GameActions.search({
```

Comment:
No need to do this nested construct. Since there's no cost or target, you can just do:
```js
gameAction: GameActions.search(context => ({
...
})
```

Reply:
fixed